php/form_fields.php

```diff
@@ -0,0 +1,141 @@
+                'default' => get_option('blockonomics_' . $currencyCode) == 1 ? 'yes' : 'no',
+            );
+            $firstItem = false;
+        }
+
+        $form_fields['advanced_divider'] = array(
+            'id'    => 'advanced_divider',
+            'type'  => 'divider'
+        );
+
+        $form_fields['extra_margin'] = array(
+            'title' => __('Advanced<p class="block-title-desc">Setting for advanced control</p>', 'blockonomics-bitcoin-payments'),
+            'type' => 'number',
+            'description' => __('Increase live fiat to BTC rate by small percent', 'blockonomics-bitcoin-payments'),
+            'subtitle' => __('Extra Currency Rate Margin %', 'blockonomics-bitcoin-payments'),
+            'default' => get_option('blockonomics_extra_margin', 0),
+            'placeholder' => __('Extra Currency Rate Margin %', 'blockonomics-bitcoin-payments')
+        );
+        $form_fields['underpayment_slack'] = array(
+            'title' => __('', 'blockonomics-bitcoin-payments'),
+            'type' => 'number',
+            'label' => __('Under Payment', 'blockonomics-bitcoin-payments'),
+            'description' => __('Allow payments that are off by a small percentage', 'blockonomics-bitcoin-payments'),
+            'subtitle' => __('Underpayment Slack %', 'blockonomics-bitcoin-payments'),
+            'default' =>  get_option('blockonomics_underpayment_slack', 0),
+            'placeholder' => __('Underpayment Slack %', 'blockonomics-bitcoin-payments')
+        );
+        $form_fields['no_javascript'] = array(
+            'title' => __('', 'blockonomics-bitcoin-payments'),
+            'type' => 'checkbox',
+            'subtitle' => __('No Javascript checkout page', 'blockonomics-bitcoin-payments'),
+            'label' => __('Enable this if you have majority customer that uses tor like browser that blocks JS', 'blockonomics-bitcoin-payments'),
+            'default' => get_option('blockonomics_nojs') == 1 ? 'yes' : 'no',
+        );
+        $form_fields['partial_payment'] = array(
+            'title' => __('', 'blockonomics-bitcoin-payments'),
+            'type' => 'checkbox',
+            'subtitle' => __('Partial Payments', 'blockonomics-bitcoin-payments'),
+            'label' => __('Allow customer to pay order via multiple payement  ', 'blockonomics-bitcoin-payments'),
+            'default' => $blockonomics->is_partial_payments_active() ? 'yes' : 'no',
+        );
+        $form_fields['network_confirmation'] = array(
+            'title' => __('', 'blockonomics-bitcoin-payments'),
+            'subtitle' => __('Network Confirmations', 'blockonomics-bitcoin-payments'),
+            'type' => 'select',
+            'description' => __('Network Confirmations required for payment to complete', 'blockonomics-bitcoin-payments'),
+            'default' => __(get_option('blockonomics_network_confirmation', 2), 'blockonomics-bitcoin-payments'),
```

Comment:
We are translating the saved value which is not required.
`'default' => __(get_option('blockonomics_network_confirmation', 2), 'blockonomics-bitcoin-payments')` should be `'default' => get_option('blockonomics_network_confirmation', 2)` as we have done for other settings.

Reply:
Same with the $callback_url. There is no need to use `__(..., 'blockonomics-bitcoin-payments')` to translate this value.